pontoon/base/static/js/lib/fluent-syntax.js

```diff
@@ -1,15 +1,31 @@
-/* fluent-syntax@0.3.0 */
+/* fluent-syntax@0.4.0 */
```

Comment:
The syntax didn't change, but the API did. See https://github.com/projectfluent/fluent.js/blob/master/fluent-syntax/CHANGELOG.md. Can you point me to where fluent-syntax is used in JS?